testsuite/tests/lib-digest/md5.ml

```diff
@@ -120,10 +120,10 @@ let string_to_data s =
   for i = 0 to 15 do
     let j = i lsl 2 in
     data.(i) <-
-      Int32.logor (Int32.shift_left (Int32.of_int (Bytes.get s (j+3) |> Char.code)) 24)
-        (Int32.logor (Int32.shift_left (Int32.of_int (Bytes.get s (j+2) |> Char.code)) 16)
-          (Int32.logor (Int32.shift_left (Int32.of_int (Bytes.get s (j+1) |> Char.code)) 8)
-                       (Int32.of_int (Bytes.get s j |> Char.code))))
+      Int32.(logor (shift_left (of_int (Bytes.get s (j+3) |> Char.code)) 24)
+        (logor (shift_left (of_int (Bytes.get s (j+2) |> Char.code)) 16)
+          (logor (shift_left (of_int (Bytes.get s (j+1) |> Char.code)) 8)
+                 (of_int (Bytes.get s j |> Char.code)))))
```

Comment:
Is there a way to format this code to make it readable?

```
let byte n = Bytes.get s (j+n) |> Char.code |> Int32.of_int in
let open Int32 in
byte 0
|> logor (shift_left (byte 1) 8)
|> logor (shift_left (byte 2) 16)
|> logor (shift_left (byte 3) 24)
```

Reply:
Readable test cases?! Whatever next!

Sorry, I'd missed this one previously, but I've changed that - far better to improve the readability of the code, than just reduce the line lengths below the maximum.